tests/e2e/e2e_test.go

```diff
@@ -580,6 +580,13 @@ var _ = Describe("odoe2e", func() {
 				runCmdShouldPass("odo push --local " + tmpDir + "/nodejs-ex")
 			})
 
+			It("should be able to push changes while showing logging", func() {
+				// Push the changes with --show-log
+				getLogging := runCmdShouldPass("odo push --show-log --local " + tmpDir + "/nodejs-ex")
+
+				Expect(getLogging).To(ContainSubstring("Building component"))
```

Comment:
The string "Building component" is present in normal too odo push. Maybe some other better way to validate the logs?
```
[mrinaldas@localhost odo]$ ./odo push
Pushing changes to component: nodejs-3-5-nodejs-tipe
 ✓  Waiting for component to start
 ✓  Copying files to component
 ✓  Building component
 ✓  Changes successfully pushed to component: nodejs-3-5-nodejs-tipe
```

Reply:
Sorry, could you clarify? Bit confused what you mean..

Reply:
I mean the normal `odo push` and `odo push --show-log` both contain the string `Building component`. So is there any other way to validate that the logs are shown?

Reply:
Without changing the output message, I don't think so, could we leave this in for now?

Reply:
No problem from my side. I will leave the decision to @amitkrout